src/sentry/models/projectplatform.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    Model, BoundedBigIntegerField, sane_repr
+)
+
+
+class ProjectPlatform(Model):
+    """
+    Tracks usage of a platform for a given project.
+    """
+    __core__ = False
+
+    project_id = BoundedBigIntegerField()
+    platform = models.CharField(max_length=64)
+    date_added = models.DateTimeField(default=timezone.now)
+    last_seen = models.DateTimeField(default=timezone.now)
```

Comment:
What about index here? Since this is used to cleanup old records during the `collect_project_platforms` task.